contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/client/UserFeignClient.java

```diff
@@ -16,4 +20,48 @@ public interface UserFeignClient {
             value = "/users"
     )
     List<User> getUsers();
+
+    @RequestMapping(
```

Comment:
Generally we do one PR per feature / issue
Here you do quite a lot, which makes the PR quite big and difficult to understand

Reply:
Can I keep this the way it's now? I'll keep in mind for the future and rename this PR